grafana.advisories.yaml

```diff
@@ -23,3 +23,13 @@ advisories:
         data:
           type: vulnerable-code-not-included-in-package
           note: Only affects Windows
+
+  - id: CVE-2019-3826
+    aliases:
+      - GHSA-3m87-5598-2v4f
+    events:
+      - timestamp: 2023-12-14T10:32:41Z
+        type: false-positive-determination
+        data:
+          type: component-vulnerability-mismatch
```

Comment:
It sounds like this is the right component but the wrong version — is that right?

If so let's try to use `vulnerable-code-version-not-used` instead for cases like this, please!

cc: @mamccorm @rawlingsj